package.json

```diff
@@ -59,6 +59,7 @@
   "engines": {
     "node": ">=8.0.0"
   },
+  "eslintIgnore": ["node_modules", "test"],
```

Comment:
?
What's the point of ignoring those if you only lint the `src/` folder anyway, test and node_modules are outside of it.

Reply:
i dunno but good practice